cmd/thor/node/metrics.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package node
+
+import (
+	"time"
+
+	"github.com/vechain/thor/v2/telemetry"
+)
+
+var (
+	metricBlockProposedCount    = telemetry.LazyLoadCounterVec("block_proposed_count", []string{"status"})
+	metricBlockProposedTxs      = telemetry.LazyLoadCounterVec("block_proposed_tx_count", []string{"status"})
+	metricBlockProposedDuration = telemetry.LazyLoadHistogramVec(
+		"block_proposed_duration_ms", []string{"status"}, telemetry.Bucket10s,
+	)
+
+	metricBlockReceivedCount        = telemetry.LazyLoadCounterVec("block_received_count", []string{"status"})
```

Comment:
Is imported sounds better than received?

Reply:
Not really.. 😄  Importing seems like it's an outbound action, received is an inbound action. 
Ah sure look it turns out to be confusing for the most of people, we can change that in the future ?